crates/workspace/src/pane_group.rs

```diff
@@ -867,7 +867,14 @@ mod element {
             debug_assert!(flexes.len() == len);
             debug_assert!(flex_values_in_bounds(flexes.as_slice()));
 
-            let magnification_value = WorkspaceSettings::get(None, cx).active_pane_magnification;
+            let mut magnification_value =
+                WorkspaceSettings::get(None, cx).active_pane_magnification;
+            if let Some(magnification) = WorkspaceSettings::get(None, cx)
+                .active_pane_modifiers
+                .magnification
```

Comment:
I think we can remove the old parameter and add a `BREAKING:` note into the release notes.

Reply:
that doesn't need any code change, right? just editing the pr description?

Reply:
Yep.

Reply:
Oh, we might to also edit `configuring-zed.md`

Reply:
done